draft-ietf-quic-tls.md

```diff
@@ -2050,7 +2059,7 @@ included. All values are shown in hexadecimal.
 
 ## Keys
 
-The labels generated during the execution of the HKDF-Expand-Label function and
+The labels (Hkdf.label) generated during the execution of the HKDF-Expand-Label function and
```

Comment:
I don't think that this is right.  I think that you mean:

```suggestion
The labels generated during the execution of the HKDF-Expand-Label function (that is, HkdfLabel.label) and part of the value
```

Reply:
Oops, yes, the HkdfLabel bit got garbled in transmission from brain to fingers.
Your suggestion looks fine; I just wanted to reiterate that it's the whole HkdfLabel.label (including the "tls13 " prefix) that we show, not just the argument to HKDF-Expand-Label.